submissions/tests/test_models.py

```diff
@@ -166,3 +178,56 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+
+class TestTeamSubmission(TestCase):
+    """
+    Test the TeamSubmission class
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = TestTeamSubmission.create_user('user1')
+        super(TestTeamSubmission, cls).setUpTestData()
+
+    def test_create_team_submission(self):
+        ts = TeamSubmission.objects.create(
```

Comment:
Nit: you could create class-level variables to store these `TeamSubmission` objects in inside of `setUpTestData`, so that they get cleaned up after all the tests in the class run.